integration-cli/docker_cli_run_test.go

```diff
@@ -4413,7 +4407,6 @@ func (s *DockerSuite) TestRunAddDeviceCgroupRule(c *testing.T) {
 
 // Verifies that running as local system is operating correctly on Windows
 func (s *DockerSuite) TestWindowsRunAsSystem(c *testing.T) {
-	testRequires(c, DaemonIsWindowsAtLeastBuild(osversion.RS3))
```

Comment:
Interesting; so this is the test that's failing 🤔

Reply:
I wonder if this test was perhaps always skipped (see code below that was removed and mentions we can of use the osversion, because the test binaries are not manifested)